.travis.yml

```diff
@@ -1,5 +1,15 @@
+sudo: required
+dist: trusty
 language: node_js
 node_js:
   - "6"
-script: npm run cover
+before_install:  
```

Comment:
can you add a comment as to what this `before_install` is doing, especially for the lines that aren't too readable